lib/colorls/core.rb

```diff
@@ -96,23 +101,25 @@ def filter_contents(path)
       end
     end
 
-    def sort_contents(path)
-      @contents.sort! { |a, b| cmp_by_dirs(path, a, b) }
+    def sort_contents
+      case @sort
+      when :time
```

Comment:
Support for `when :size` too, please? 😃

Reply:
Ignore the above comment. Size support will be added by @neshmi, as assigned in issue #152